### DISCUSSION TITLE
#14. VATZ biweekly meeting at `2022-08-03 08:00:00 -06:00:00`

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#200] [General]
### 1. Overall
![image](https://user-images.githubusercontent.com/6308023/180569541-2751bfb7-c5a0-4d11-b882-dcaa1f1b3acd.png)

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
12 | 4/16 | 25 
13|  7/20 | 35

> Thanks for your hard work for 2 weeks.
> The target progress rate for each sprint is 40% to 55%. 

---
- Finished
  - #151 
  - #157
  - #163
  - #167 
  - #168 
  - #179
  - #181
  
- Move to Sprint 13
  - #85 
  - #87 
  - #119 
  - #121 
  - #127 
  - #146 
  - #174 
  - #175 
  - #176 
  - #178 
  - #187 

---

Discussion:
- https://github.com/dsrvlabs/vatz/issues/88

**Comment by xellos00** [comment]
# 동요

- https://github.com/dsrvlabs/vatz/issues/199
- #211 
- #217

**Comment by rootwarp** [comment]
# Joonkyo

- https://github.com/dsrvlabs/vatz/issues/208
- https://github.com/dsrvlabs/vatz/issues/213
- #220 

---
- https://github.com/dsrvlabs/vatz/pull/209
- https://github.com/dsrvlabs/vatz/pull/214

**Comment by meetrick** [comment]
# 황제
- no update

**Comment by heejin-github** [comment]
# 희진
- #206
   - #215
- #118

**Comment by gnongs** [comment]
## 진홍
- #127

**Comment by kim201212** [comment]
# 형기

- https://github.com/dsrvlabs/vatz/issues/196

**Reply by xellos00** [reply]
TODO
- Github action for go lint 
- Get a secret for VATZ (action purpose only)

**Comment by xellos00** [comment]
## Way to Start Official Plugins

1. `make start with script`
---
Bin -> each scripts
- Build
- Start 
- Stop
- Clean 
Make 

---

for current meeting.
1. We do not want to have many official plugins
2. our end-user will be who is more advanced than our knowledges
3. We need to narrow down on selection. 
4. Let's settle official plugins first and then apply above.
